README.md

```diff
@@ -7,6 +7,202 @@
 [![Coverage Status](https://coveralls.io/repos/github/voxpupuli/puppet-webhook/badge.svg?branch=master)](https://coveralls.io/github/voxpupuli/puppet-webhook?branch=master)
 [![Dependency Status](https://gemnasium.com/badges/github.com/voxpupuli/puppet-webhook.svg)](https://gemnasium.com/github.com/voxpupuli/puppet-webhook)
 
-The puppet webhook server is a Sinatra-based application for receiving webhooks that will trigger r10k runs on your puppet master.
+## What is puppet_webhook
 
-**NOTE: This is very much still in development and NOT stable**
+puppet_webhook is a Sinatra-based application receiving REST-based calls to trigger Puppet and r10k-related tasks such as:
+
+* Webhooks from Source Code systems to trigger r10k environment and module deploys
+* REST calls from systems to trigger Puppet Decommissions such as:
+    * `puppet node clean`
+    * `puppet cert clean`
+    * `puppet cert revoke`
+    * etc.
+* Send notifications via Slack
+
+## Prerequisites
+
+* Ruby 2.1.9 or greater
+* Puppet 4.7.0 or greater
```

Comment:
Is this a hard requirement or only listed because Vox Pupuli supports it? If so, we should bump it to 4.7.1, that is the lowest version we should support in our modules.

Reply:
Will do